File: api/Controllers/Models/Entities/Product_Rating.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace IPKP___API.Controllers.Models.Entities
{
  public class Product_Rating
  {
    [Key]
    public Guid Product_Rating_ID { get; set; }
    public virtual Customer Customer_ID { get; set; }
    public virtual Stock_Item Stock_Item_ID { get; set; }
    public int Product_Star_Rating { get; set; }
    [StringLength(255)]
    public string Product_Rating_Comments { get; set; }
  }
}

File: customer/src/app/Models/productrating.ts
import { Customer } from "./customer"
import { Stock_Item } from "./stockitem"

export class ProductRating{
    product_Rating_ID!: string
    //customer!: Customer
    customer_ID!:String;
    stock_Item!: Stock_Item
    stock_Item_ID!: string
    product_Star_Rating!: Number 
    product_Rating_Comments!: String
}